hudi-common/src/main/java/org/apache/hudi/avro/ConvertingGenericData.java

```diff
@@ -128,7 +125,8 @@ public boolean validate(Schema schema, Object datum) {
       case LONG:
         return isLong(datum)
             || TIME_MICROS_CONVERSION.getConvertedType().isInstance(datum)
-            || TIMESTAMP_MICROS_CONVERSION.getConvertedType().isInstance(datum);
+            || TIMESTAMP_MICROS_CONVERSION.getConvertedType().isInstance(datum)
+            || LOCAL_TIMESTAMP_MICROS_CONVERSION.getConvertedType().isInstance(datum);
```

Comment:
Should `TIME_MILLIS_CONVERSION`, `LOCAL_TIMESTAMP_MILLIS_CONVERSION`, and `TIMESTAMP_MILLIS_CONVERSION` also be handled or they are handled somewhere else?